presto-main/src/main/java/com/facebook/presto/execution/BlackholeWarningCollector.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.execution;
+
+import com.facebook.presto.spi.PrestoWarning;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+public class BlackholeWarningCollector
```

Comment:
How do you feel about naming this as `NoOpWarningCollector`? Whenever I see the name "blackhole" it reminds me the blackhole connector. Also we have implementations with similar naming convention, such as `NoOpResourceGroupManager`, `NoOpSessionSupplier`, etc.

Reply:
Make sense.